client/src/components/EventView/RSVPCard/index.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
You can revert changes for this file. The component already exists and is called "ActionPrompt"